src/tweenable.rs

```diff
@@ -74,10 +74,14 @@ pub enum TweenState {
 /// complete cycle start -> end -> start counts as 2 iterations and raises 2
 /// events (one when reaching the end, one when reaching back the start).
 ///
+/// If the tick delta is high enough that multiple completions have occurred in
```

Comment:
Maybe we should add a `completion_count` parameter to the even then, to report how many times a loop was completed during a single `tick()`?

Reply:
I think that's a reasonable improvement, but I'd personally rather not add anything until someone comes along with an actual use case. Otherwise it's just cruft that needs to be maintained without any clear users.